api/src/basic.rs

```diff
@@ -60,6 +65,56 @@ impl BasicApi {
         Html(OPEN_API_HTML.to_string())
     }
 
+    /// Show some basic info of the node.
+    #[oai(
+        path = "/info",
+        method = "get",
+        operation_id = "info",
+        tag = "ApiTags::General"
+    )]
+    async fn info(&self) -> Json<HashMap<String, String>> {
```

Comment:
Will it be a breaking change if later we change this to a nested structure? 
(How does it look like in the API spec?)

Reply:
Generated the spec, PTAL

Reply:
Looks like it just says "json in text" (so we are good)?
But can an expert confirm? 😂  @banool ?